src/classes/ADDR_AdminAcc_TEST.cls

```diff
@@ -0,0 +1,891 @@
+        // now let's update the default Address for both accounts
+        addrs[0].MailingStreet__c = 'New Street';
+        addrs[0].MailingCity__c = 'New City';
+        addrs[1].MailingStreet__c = 'New Street';
+        addrs[1].MailingCity__c = 'New City';
+
+        ADDR_Addresses_TDTM.hasRunAddrTrigger = false;
+        Test.startTest();
+        update addrs;
+        Test.stopTest();
+        
+        map<Id, Account> mapAccIdAcc = new map<Id, Account>([select Id, Name, BillingStreet, BillingCity, BillingState, 
+        BillingPostalCode, BillingCountry, BillingLatitude, BillingLongitude, is_Address_Override__c, Current_Address__c from Account]);
+        
+        for (Account acc : mapAccIdAcc.values()) {
+            // verify that the Account address is override
+            if (acc.is_Address_Override__c) { 
+                system.assert(!acc.BillingStreet.contains('street override'));
+                system.assert(!acc.BillingCity.contains('city override'));
+                system.assertEquals(acc.Current_Address__c, addrNew.Id);            
+            // verify that the Account address is new
+            } else { 
+                system.assert(acc.BillingStreet.contains('New Street'));
+                system.assert(acc.BillingCity.contains('New City'));
+                system.assertNotEquals(null, acc.Current_Address__c);
+            }
+        }
+    }
+
+    /*********************************************************************************************************
+    @description Insert new default addresses to Account w/ existing default addresses  
+    verify:
+        Account address matches new default
+        old default addresses no longer marked default
+    **********************************************************************************************************/            
+    @isTest
+    public static void insertNewDefaultAddr() {
+        configSettings();
+          
+        // this creates a default Address for each Account
+        UTIL_UnitTestData_TEST.AccsWithAddrs accsAddrs = UTIL_UnitTestData_TEST.createTestAccsAddrs(2, UTIL_Describe.getAdminAccRecTypeID());
+        List<Address__c> addrs = accsAddrs.addrs;
+        
+        // create additional addresses
+        addrs = UTIL_UnitTestData_TEST.initTestAddr(2);
+        for (Integer i = 0; i < accsAddrs.accs.size(); i++) {
+            addrs[i].Parent_Account__c = accsAddrs.accs[i].Id;
+            addrs[i].Default_Address__c = true;
+            addrs[i].MailingStreet__c = 'New Default Street' + i;
+            addrs[i].MailingCity__c = 'New Default City' + i;
+        }
+        ADDR_Addresses_TDTM.hasRunAddrTrigger = false;
+        Test.startTest();
+        insert addrs;
+        ADDR_Addresses_TDTM.hasRunAddrTrigger = false;        
+        Test.stopTest();
+
+        // verify that the Account addresses are new
+        map<Id, Account> mapAccIdAcc = new map<Id, Account>([select Id, Name, BillingStreet, BillingCity, BillingState, 
```

Comment:
If the table in this query contains more than 50'000 records a runtime exception will be thrown. Consider either introducing a `WHERE` clause, or adding a `LIMIT` to contain the number of records returned.